lib/gooddata/lcm/user_bricks_helper.rb

```diff
@@ -0,0 +1,25 @@
+module GoodData
+  module LCM2
+    class UserBricksHelper
+      class << self
+        def resolve_client_id(domain, project)
+          md = project.metadata
```

Comment:
Refers to 'project' more than self (maybe move it to another class?) (FeatureEnvy)